src/components/hand-controls.js

```diff
@@ -244,9 +253,11 @@ module.exports.Component = registerComponent('hand-controls', {
     if (animationActive) { mesh.play(animationActive, 0); }
 
     // play new animation.
-    mesh.mixer.clipAction(animation).loop = 2200;
-    mesh.mixer.clipAction(animation).clampWhenFinished = true;
-    mesh.mixer.clipAction(animation).timeScale = timeScale;
+    clipAction = mesh.mixer.clipAction(animation);
+    if (!clipAction) { return; }
```

Comment:
I prefer this to break loudly on a Q/A pass. If the animation is not present then `hand-controls` are not working as expected and I want it to be hard to miss. The hand model and this component will be always in sync.